notebook/static/tree/js/notebooklist.js

```diff
@@ -520,6 +545,12 @@ define([
             .addClass("item_name")
             .appendTo(link);
 
+        $("<span/>")
+            .addClass("file_size")
+            .addClass("pull-right")
+            .css("width", "65px")
```

Comment:
You likely want to avoid inline css.

Reply:
do you know where the css should go? from what i've seen, most of the CSS is defined like this:

https://github.com/jupyter/notebook/blob/8034d714298e212eaef43b8e43f04d55c87bed20/notebook/static/tree/js/notebooklist.js#L1250-L1263

Reply:
Likely in `notebook/static/tree/less/tree.less` there are case when we do inline css, there might be a reason for progress-bar, maybe not, if not that's likely technical dept we need to fix at some point.

You'll  likely need to recompile thee css with `python setup.py css` after modification of the less file.